mkinitcpio.d/example.preset

```diff
@@ -17,9 +18,11 @@ ALL_config='/etc/mkinitcpio.conf'
 #default_kver="3.0-ARCH"
 #default_config="/etc/mkinitcpio.conf"
 default_image="/tmp/initramfs-linux.img"
+default_efi_image="/efi/EFI/Linux/linux.efi"
```

Comment:
This should perhaps be namespaced to avoid messing with foreign linuxes under EFI/

Reply:
Namespaces how? `/efi/EFI/Arch/linux.efi` or `/efi/EFI/Linux/arch-linux.efi`?

Reply:
I haven't come up with any implementation details; The latter sounds fine to me, if the former is not an option. I just think that we should have _some_ kind of namespacing, else we might clobber existing stuff from elsewhere, the chances of which I'd like to minimize.